VCS/0. Overview/1. 服務概觀.md

```diff
@@ -1 +1,40 @@
+---
+tags: Guide, VCS, TW
+title: 服務概觀 | en
+GA: UA-155999456-1
+---
 
+{%hackmd @docsharedstyle/default %}
+{%hackmd @docsharedstyle/twccheader-en %}
+
+# <img class="icon" src="https://cos.twcc.ai/SYS-MANUAL/uploads/upload_af58322eb82b649d1f29aca1f201a117.png" width="25" height="25"> Service overview
+
+Virtual Compute Service Instance is a traditional general host rental service provided by TWCC. It can be quickly deployed and recycled through automated tools and processes and have the ability to mount multiple virtual disks at the same time, create data disks, and freely and flexibly allocate instance storage space. It is the best choice for both intelligent calculation and cost-saving.
+
+## Benefits
+
+#### <img src="https://cos.twcc.ai/SYS-MANUAL/uploads/upload_fdddc2308f91f4689d5765a35b457c72.png" width="35" height="25"> Built on the Cloud with Many Options
```

Comment:
[SUG]

Built on the Cloud with Many Options > 

Built on the cloud with many options